apps/acrs/behaviours/parent-summary.js

```diff
@@ -0,0 +1,26 @@
+const moment = require('moment');
+module.exports = superclass => class extends superclass {
+  configure(req, res, next) {
+    if(req.sessionModel.get('referred-parents') && !req.sessionModel.get('referred-parents').aggregatedValues.length) {
+      req.form.options.addStep = 'parent';
+    }
+    super.configure(req, res, next);
+  }
+
+  locals(req, res) {
```

Comment:
A little commentary here would go a long way. This function provides a very useful facility for decorating the summary page output and an indication of how it iteracts with the items array and template configuration would be handy.

Reply:
Have added some commentary to this and a couple of other new behaviours.